setup/jest.config.js

```diff
@@ -132,7 +132,6 @@ module.exports = {
     'src/modules/common/components/timestamp/index.js',
     'src/utils/localJSONStorage.js',
```

Comment:
NRTTL
I can still get duplicate name using edit function for bookmarks and network (edit account is fine though)

Reply:
Fixed